cobbler/modules/sync_post_wingen.py

```diff
@@ -577,8 +583,8 @@ def gen_win_files(distro: "Distro", meta: Dict[str, Any]):
 
         distro = profile.get_conceptual_parent()  # type: ignore
 
-        if distro and distro.breed == "windows":
+        if distro and distro.breed == "windows":  # type: ignore[reportUnknownMemberType]
             logger.info("System: %s", system.name)
             meta = utils.blender(api, False, system)
```

Comment:
@tpw56j Could you confirm this? I think the line below is actually a bug and we instead wanted to use `meta` there since in `gen_win_files()` we need to access the kernel path which is not in `autoinstall_meta` afaik.

Reply:
Yes, it is a bug. There is a missing lines in this code.
The code should look like this:
```python
         if distro and distro.breed == "windows":
             logger.info("System: %s", system.name)
             meta = utils.blender(api, False, system)
 +           autoinstall_meta = meta.get("autoinstall_meta", {})
 +           meta.update(autoinstall_meta)
             gen_win_files(distro, autoinstall_meta)
```

Reply:
Thanks for the quick answer. I will include this in a push to this PR.